modules/apps/transfer/keeper/keeper.go

```diff
@@ -77,6 +77,11 @@ func NewKeeper(
 	}
 }
 
+// WithICS4Wrapper sets the ICS4 wrapper
```

Comment:
can we add a little bit more information on when it would be necessary to call this function in this godoc?

Reply:
done